openid-sharedsignals-framework-1_0.md

```diff
@@ -1462,9 +1452,9 @@ Errors are signaled with HTTP status codes as follows:
 
 Examples:
 
-1. If a Receiver makes a request with an invalid OAuth token, then the
+1. If a Receiver makes a request with an invalid token, then the
```

Comment:
Since we are dropping references to OAuth, does it make sense to change this sentence to: "If a Receiver makes a request with improper authorization, then..." (i.e. not assume token based authorization)